README.md

```diff
@@ -1,19 +1,14 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse
 
-> Description the project.
+> This project is to help learn the Github flow.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+- HTML, Css
+ 
 
 
 ## Getting Started
```

Comment:
- [OPTIONAL] I can see you are using the Microverse template, Great Job 👌, but you forgot to customize the `Getting Started` section, please update it.

> The `Getting Started` section will be helpful for developers and recruiters to launch your project.